listenbrainz_spark/recommendations/candidate_sets.py

```diff
@@ -35,7 +35,12 @@ def get_listens_for_rec_generation_window():
     df = None
     end_date = datetime.utcnow()
     begin_date = stats.adjust_days(end_date, -config.RECOMMENDATION_GENERATION_WINDOW)
```

Comment:
@mayhem  If everything looks okay, should I introduce a check for config.REOMMENDATION_GENERATION_WINDOW (if not less than 28 abort)